.gitignore

```diff
@@ -16,3 +16,7 @@ dist/
 declarations/
 
 .mops
+
+# build files
+build/
+out/
```

Comment:
We need this directory. Otherwise it will create permission problems if docker creates this directory itself. The template repo has the out/.gitkeep file just for this purpose. This file must be checked in. 

But you are right that we can ignore other files inside the out directory (like the .wasm and .wasm.gz in there)

Reply:
Fixed. Now git ignores everything under this directory but `.gitkeep` file